website/docs/reference/operators/query/logical-operators/and.mdx

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 1
+description: Combines multiple query expressions and outputs documents that match all expressions.
+---
+
+# `$and`
+
+import CodeBlock from '@theme/CodeBlock';
+import Dataset from '!!raw-loader!./logical-operators/2-insert-dataset.request.js';
+import AndOperatorRequest from '!!raw-loader!./logical-operators/3-and-operator.request.js';
+import AndOperatorResponse from '!!raw-loader!./logical-operators/3-and-operator.response.js';
+
+The `$and` operator combines multiple query expressions and returns documents that match all the expressions.
+
+The operator accepts an array of query expressions as shown below, and all expressions must be satisfied for a document to be included in the result set.
+
+```js
+db.runCommand({
+  find: '<collection_name>',
+  filter: {
+    $and: [
+      { <query-expression1> },
```

Comment:
Let's use either `_` or `-` but not mix them like `<collection_name>` and `<query-expression1>`